lib/phenix.rb

```diff
@@ -3,23 +3,36 @@
 
 module Phenix
   class << self
-    attr_accessor :database_config_path, :schema_config_path
+    attr_accessor :database_config_path, :schema_path, :skip_database
     attr_accessor :current_configuration
 
     def configure
-      self.database_config_path = File.join(Bundler.root, 'test', 'database.yml')
-      self.schema_config_path   = File.join(Bundler.root, 'test', 'schema.rb')
+      self.database_config_path = File.join(Dir.pwd, 'test', 'database.yml')
+      self.schema_path          = File.join(Dir.pwd, 'test', 'schema.rb')
+      self.skip_database        = nil
 
       yield(self) if block_given?
     end
   end
 
+  def rise!(with_schema: true, config_path: Phenix.database_config_path)
+    load_database_config(config_path)
+    drop_databases
+    with_schema ? create_and_populate_databases : create_databases
```

Comment:
I could pass `with_schema` to `create_database` and have `populate_database if with_schema`